uk.co.compucorp.civicrm.hrleaveandabsences/js/test/leave-type-wizard/components/leave-type-wizard.component.spec.js

```diff
@@ -239,6 +239,45 @@ define([
           });
         });
       });
+
+      describe('validators', function () {
+        var sampleField;
+        var sampleErrorMessage = 'Invalid format';
+
+        beforeEach(function () {
+          sampleField = controller.fieldsIndexed.default_entitlement;
+          sampleField.validations = [
+            {
+              rule: /^\d+$/,
+              message: sampleErrorMessage
+            }
+          ];
+        });
+
+        describe('when user enters a value in a wrong format', function () {
+          beforeEach(function () {
+            sampleField.value = 'Not a number';
+
+            $rootScope.$digest();
+          });
+
+          it('sets the error to the field', function () {
+            expect(sampleField.error).toBe(sampleErrorMessage);
+          });
+
+          describe('when user enters a value in a valid format', function () {
```

Comment:
Should not this `describe` block be outside of the `describe` block in line 257?

Reply:
No, it tests like that intentionally, to check if user can **fix** the error. I renamed to `'when user changes the value to a valid format'`